tools/manifest/sourcefile.py

```diff
@@ -723,7 +723,14 @@ def testdriver_nodes(self) -> List[ElementTree.Element]:
         """List of ElementTree Elements corresponding to nodes representing a
         testdriver.js script"""
         assert self.root is not None
-        return self.root.findall(".//{http://www.w3.org/1999/xhtml}script[@src='/resources/testdriver.js']")
+        # `xml.etree.ElementTree.findall` has a limited support of xPath, so
+        # explicit filter is required.
+        return [node for node in
+                self.root.findall(".//{http://www.w3.org/1999/xhtml}script")
+                if node.attrib.get('src',
+                                   "") == '/resources/testdriver.js' or
+                node.attrib.get('src', "").startswith(
+                    '/resources/testdriver.js?')]
```

Comment:
Wouldn't it be simpler to combine the two conditions in one, like `if node.attrib.get('src', "").startswith('/resources/testdriver.js')]`?

Reply:
I want to avoid getting nodes like `/resources/testdriver.js.SOME_SUFFIX?...`. I can imagine there something like `/resources/testdriver.js.vendor_extension?some_query_params`.

Even though the lint rule above will fail on `/resources/testdriver.js.vendor_extension?some_query_params`, there can be valid use cases for it. If so, the user will have to disable the failing lint rule for the specific file.

Reply:
Fair enough, thanks.